server/admin/logstream.go

```diff
@@ -0,0 +1,145 @@
+package admin
+
+import (
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/navidrome/navidrome/log"
+	"github.com/navidrome/navidrome/model"
+	"github.com/navidrome/navidrome/model/request"
+	"github.com/sirupsen/logrus"
+)
+
+// LogEntry represents a log entry in the format sent to the UI
+type LogEntry struct {
+	Time    time.Time              `json:"time"`
+	Level   string                 `json:"level"`
+	Message string                 `json:"message"`
+	Data    map[string]interface{} `json:"data"`
+}
+
+type Router struct {
+	http.Handler
+	ds            model.DataStore
+	authenticator func(ds model.DataStore) func(http.Handler) http.Handler
+	jwtRefresher  func(http.Handler) http.Handler
+	requireAdmin  func(http.Handler) http.Handler
+}
+
+func NewRouter(ds model.DataStore, authenticator func(ds model.DataStore) func(http.Handler) http.Handler,
```

Comment:
Can't we leverage the current `server/events` package instead of introducing a new SSE endpoint?